services.go

```diff
@@ -34,7 +34,7 @@ type services struct {
 
 func (c *Client) initServices() {
 	c.Argus = argus.NewService(c)
-	c.Costs = costs.NewService(c)
+	c.Costs = costs.NewService(c, c.config.BaseUrl)
```

Comment:
```suggestion
	c.Costs = costs.NewService(c, nil)
```
seems that it needs to be nil in order for the URL to be determined by the environment

Reply:
We would like to leverage the `BaseURL` option from the client config. So we pass it along; if the option is provided; we actually ignore the settings from the environment. So it is the other way around. We considered using env vars for the change, but actually it would be nice if the config's `BaseURL` option could be used.